src/components/view/Garage/Track/track.ts

```diff
@@ -0,0 +1,162 @@
+import './track.css';
+import ICar from '../../../controller/ICar';
+
+class Track {
+    amountOfCars: number;
+    selectedID: number;
+    amountOfPages: number;
+    currentPage: number;
+    garage: ICar[];
```

Comment:
Track component is in view folder, but it's actually not a view layer, it has data and controller logic